package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ndustrial/contxt-sdk",
-  "version": "0.0.32",
+  "version": "0.0.33",
```

Comment:
Regarding updating the numbers here and in the lock file. I'm assuming you just did that in an editor and didn't run `npm version x.x.x` or whatever. Take a look at the [Contributing and Publishing](https://github.com/ndustrialio/contxt-sdk-js#contributingpublishing) section in the README file. We do it a little differently so these shouldn't actually be updated like this.

Reply:
Yeah, Greg mentioned that as well. Will do going forward.